src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -1,6 +1,8 @@
 import {
   mdiChartBoxOutline,
   mdiClose,
+  mdiArrowExpand,
+  mdiArrowCollapse,
```

Comment:
I wonder if these would be the right icons to use.  Maybe something more familiar in this location would be `mdi:window-maximize` and `mdi:window-restore`?

Reply:
My hesitation with maximize is that that's not really what this button does. I usually associate maximize with filling the screen, but this button just makes the dialog somewhat wider. 

Also the maximize icon is just a plain square, so kind of a bit boring and not that informative.

Reply:
I disagree it wouldn't be informative.  It doesn't need to be sexy; it needs to convey the function to the user.  A square at the top of a window is easily recognized as maximizing the size.  I don't think it matters if the maximum is not full screen.  Expand/collapse icons on the other hand are widely used for show/hide functionality, which is not what this is.  My 🪙🪙 at least.

Reply:
Sure no problem. I don't mind changing it, lets wait and see if this collects any further opinions.